influxql/INFLUXQL.md

```diff
@@ -44,56 +44,123 @@ decimal_digit       = "0" .. "9" .
 
 Database names are more limited than other identifiers because they appear in URLs.
 
+The rules:
+
+- must start with an upper or lowercase ASCII letter
+- may contain only ASCII leters, decimal digits, "_", and "-"
+
 ```
 db_name             = ascii_letter { ascii_letter | decimal_digit | "_" | "-" } .
 ```
 
+#### Examples:
+
+```
+mydb
+MyDB
+my-db_3
+```
+
 ## Identifiers
 
+Identifiers are things like measurement names, retention policy names, tag keys, etc.
+
+The rules:
+
+- double quoted identifiers can contain any unicode character other than a new line
+- unquoted identifiers must start with an upper or lowercase ASCII character
+- unquoted identifiers may contain only ASCII letters, decimal digits, "_", and "."
```

Comment:
Identifiers get read together with their dots and then broken up post-parsing. What @dgnorton said above is correct (except the dots in between are thrown out when split).